containertools/cmd/nbwatch/main.go

```diff
@@ -33,7 +34,31 @@ func run() error {
 	}
 	defer w.Close()
 
-	w.Add("/content/src")
+	const contentDir = "/content"
+
+	// NOTE: Watch is non-recursive.
```

Comment:
so it would watch `/content/src` but not `/content/src/my-dir/sam.ipynb`?

Reply:
Yep